packages/next/next-server/server/image-optimizer.ts

```diff
@@ -366,10 +374,14 @@ function sendResponse(
   req: IncomingMessage,
   res: ServerResponse,
   contentType: string | null,
-  buffer: Buffer
+  buffer: Buffer,
+  isStatic: boolean
 ) {
   const etag = getHash([buffer])
-  res.setHeader('Cache-Control', 'public, max-age=0, must-revalidate')
+  res.setHeader(
+    'Cache-Control',
+    isStatic ? 'immutable' : 'public, max-age=0, must-revalidate'
```

Comment:
Let's get `immutable` support added to Chrome, see https://bugs.chromium.org/p/chromium/issues/detail?id=611416